### ISSUE TITLE
Upgrade babel dependencies to rc versions

### ISSUE THREAD

**Issue by Mati365** [babel/babel-eslint#670]
after installing babel-eslint my yarn.lock contains duplicated @babel/ packages
```"@babel/code-frame@7.0.0-beta.52":
  version "7.0.0-beta.52"
  resolved "http://verdaccio.cdd.htp.iadm:4873/@babel%2fcode-frame/-/code-frame-7.0.0-beta.52.tgz#192483bfa0d1e467c101571c21029ccb74af2801"
  dependencies:
    "@babel/highlight" "7.0.0-beta.52"

"@babel/code-frame@7.0.0-rc.1":
  version "7.0.0-rc.1"
  resolved "http://verdaccio.cdd.htp.iadm:4873/@babel%2fcode-frame/-/code-frame-7.0.0-rc.1.tgz#5c2154415d6c09959a71845ef519d11157e95d10"
  dependencies:
    "@babel/highlight" "7.0.0-rc.1"
````

primary due to babel eslint packages. Babel-eslint entry:

```babel-eslint@^9.0.0-beta.3:
  version "9.0.0-beta.3"
  resolved "http://verdaccio.cdd.htp.iadm:4873/babel-eslint/-/babel-eslint-9.0.0-beta.3.tgz#229e36419ca5f20a1548cf5cd2413aa861193649"
  dependencies:
    "@babel/code-frame" "7.0.0-beta.52"
    "@babel/parser" "7.0.0-beta.52"
    "@babel/traverse" "7.0.0-beta.52"
    "@babel/types" "7.0.0-beta.52"
    eslint-scope "3.7.1"
    eslint-visitor-keys "^1.0.0"```

**Comment by edmorley** [comment]
Fixed by #668 (merged but needs a new release).

**Comment by sibelius** [comment]
fixed on 9.0.0, we can close this
